front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/pipeline/PipelineTemplate.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.front50.model.pipeline;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.spinnaker.front50.model.Timestamped;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class PipelineTemplate extends HashMap<String, Object> implements Timestamped {
+
+  @JsonIgnore
+  @SuppressWarnings("unchecked")
+  public List<String> getScope() {
```

Comment:
I debated on this a little bit, then decided to just throw something out there. I suspect that most people will scope templates to a single item (global or a specific app), which is why I chose the singular. Definitely happy to make it plural, since it is a list.